ethereum/api/src/main/java/org/hyperledger/besu/ethereum/api/jsonrpc/internal/methods/engine/EngineNewPayload.java

```diff
@@ -160,8 +164,7 @@ public JsonRpcResponse syncResponse(final JsonRpcRequestContext requestContext)
     final var block =
         new Block(newBlockHeader, new BlockBody(transactions, Collections.emptyList()));
 
-    if (mergeContext.isSyncing()
-        || mergeCoordinator.getOrSyncHeaderByHash(newBlockHeader.getParentHash()).isEmpty()) {
+    if (mergeContext.isSyncing() || parentHeader.isEmpty()) {
```

Comment:
This has a side effect of *not* appending the parent header to a backward sync if it is not in the blockchain already, since `parentHeader` is just an optional fetched from the blockchain as opposed to trying to fetch and appends the hash to backward sync if it was not found.  I think we need to keep that behavior  - perhaps we change the lookup on line 152 to use `mergeCoordinator.getOrSyncByHash`?

Reply:
as discussed, I am 👌  to rely on the append of the newPayload hash